src/components/data-management/NewProjectModal.jsx

```diff
@@ -49,7 +49,7 @@ const NewProjectModal = (props) => {
 
   const submit = () => {
     // TO-DO: this fails to clear the input field
-    // We prpbably can fix this taking advantage of
+    // We probably can fix this taking advantage of
     // the Form properties.
```

Comment:
Let's not leave TO-DOs inside the code. Better to list tasks to do in Jira.

Reply:
I think I like better having both of them 🤔 
Upaded https://biomage.atlassian.net/browse/BIOMAGE-1018 with the comment above and removed it in the code